build/DependencyVersions.props

```diff
@@ -4,7 +4,6 @@
     <CLI_SharedFrameworkVersion>2.0.0</CLI_SharedFrameworkVersion>
     <CLI_MSBuild_Version>15.3.409</CLI_MSBuild_Version>
     <CLI_Roslyn_Version>2.3.2-beta1-61921-05</CLI_Roslyn_Version>
-    <CLI_Roslyn_Satellites_Version>2.3.0-pre-20170624-6</CLI_Roslyn_Satellites_Version>
```

Comment:
Removing this is gated on getting roslyn to build its own satellites assemblies in the open and delivering them with the rest of the roslyn bits via a .nupkg out of the roslyn repo. If we remove this before that happens, then localization of the compiler will regress. The plan is to get roslyn on to the OSS localization process in 15.5, but I don't want to delete the interim process here until that happens in case we need to ship these CLI bits first.